1_space_odyssey/js/simen-9000/app.js

```diff
@@ -85,7 +85,22 @@
         b. ensure to choose the correct values and datatypes
         c. the droid should be built in your likeness (a human, with human features, 2 arms, legs etc)
     */
-    var isAlive = true;
+   /*
+ 7. Android Object REFACTOR:::::
+
+ Level 1::::::::::::::
+ Refactor (change) all the variables you have in place that initializes the android into an android object.
+ - Your android needs a head, upperBody, lowerBody & [ anyThingElseYouCanDreamOf ].
+ - Feel free to create any new properties that will further describe your android [ anyThingYouCanDreamOf ].
+ - next update your application to use this newly refactored code
+
+ tip: the console.log that prints the variables of the android will need to be updated
+
+ Level 2::::::::::::::
+ Refactor the planets array to be an array of objects, each object must be its own planet.
+  - next update your application to use this newly refactored code
+*/
+    /* var isAlive = true;
```

Comment:
you can delete thes commented out variables now. you have successfully refactored your code and this is DEAD code now.